ddev/src/ddev/e2e/agent/docker.py

```diff
@@ -9,6 +9,8 @@
 from functools import cache, cached_property
 from typing import TYPE_CHECKING, Callable
 
+from tenacity import retry, stop_after_attempt, wait_fixed
```

Comment:
nit: do you mind if we use this library instead that is a nicer wrapper around that? https://github.com/hynek/stamina

Reply:
Or if you want you could merge this and then open up another PR which replaces all uses of that library with the wrapper

Reply:
I switched to `stamina` for `ddev`. For now, this is the only place where we have a retry in the new version of `ddev`, so I did not have to modify anything else. I think we could move from `tenacity` to `stamina` at the same time as we migrate the `ddev` command. Would that make sense?